01/components/ClockTime.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+// eslint-disable-function-component-definition 
```

Comment:
Można ustawić w ogólnej konfiguracji zamiast pisać wszędzie to samo :P

PS. Tak na następny raz ;)